lua/ttt2/libraries/door.lua

```diff
@@ -54,7 +54,7 @@ local valid_doors = {
 local function GetClosedAngle(ent)
 	local data = ent:GetInternalVariable("m_angRotationClosed")
 
-	if not data then return end
+	if not data or #data < 3 then return end
```

Comment:
To keep it save, pls add `istable()` around the first data check :)

Reply:
I have to check if it returns true, because it isn't a table but userdata

Reply:
oh okay. Then it should be fine. Thought bc indexing e.g. a number would lead to an error

Reply:
Tested it just now and no, userdata isn't counted as a table